samples/extensions/khr/nbody/main.cpp

```diff
@@ -216,8 +214,8 @@ void NBody::initializeGL()
         return program;
     };
 
-    vertex_shader = create_shader("./nbody.vert.glsl", GL_VERTEX_SHADER);
-    fragment_shader = create_shader("./nbody.frag.glsl", GL_FRAGMENT_SHADER);
+    vertex_shader = create_shader("nbody.vert.glsl", GL_VERTEX_SHADER);
+    fragment_shader = create_shader("nbody.frag.glsl", GL_FRAGMENT_SHADER);
```

Comment:
Minor, will this `./filename.cl` syntax still work?  Or is it just unnecessary now, so no reason to include it?

Reply:
It should. The implementation gets the absolute path to the executable's folder and appends a `/` unconditionally before concatenating it with the user-provided string. `"/home" + "/" + "./kernel.cl"` should work. It's not a normalized path, but it evaluates to the same thing without the `./`